src/averageAges.js

```diff
@@ -20,6 +20,22 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const manArr = people
+    .filter((person) => {
```

Comment:
always try to use destructurization whenever it's possible
```suggestion
    .filter(({ died, ... }) => {
```